KrispyTech/airneis
# Airneis 109 mobile translations

# DONE 

- Set up translations for mobile
- Translated all screens, components, and errors
- Created a locale selector I put on home page

# TO TEST

- Launch the app (don't forget the bun i to install the translations package)
-  On home screen you should see the locale selector
### Folded version
![image](https://github.com/KrispyTech/airneis/assets/117679829/0ac3144c-aa39-43f9-96da-1f9f2a10eca9)
# Unfold version
![image](https://github.com/KrispyTech/airneis/assets/117679829/86f8c3b7-c610-48aa-ac14-be998192109a)

- Check that everything is translated
